plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                        type: int
+                      prepend_autonomous_system_count:
+                        description:
+                        - Artificially increase the AS path length when advertising routes via BGP.
+                        - Makes a route appear less favorable to external peers.
+                        - prepend_autonomous_system_count should be between 1 to 10.
+                        - If prepend_autonomous_system_count is not set, the default value is 0.
+                        - prepend_autonomous_system_count can be updated.
+                        type: int
+                  layer3_handoff_ip_transit:
+                    description:
+                    - Adds layer 3 handoffs with ip transit in fabric devices.
+                    - Configured when IP traffic is routed from the SDA fabric to external networks.
+                    - If layer3_handoff_ip_transit is set, border type will be considered as Layer3.
+                    type: list
+                    elements: dict
+                    suboptions:
+                      transit_network_name:
+                        description:
+                        - Network that connects multiple SDA fabrics or networks.
+                        - Mandatory parameter for all operations in L3 Handoff with IP transit.
+                        - transit_network_name cannot be updated.
+                        type: str
+                      interface_name:
+                        description:
+                        - Refers to the specific network interface in the border device.
+                        - Mandatory parameter for all operations in L3 Handoff with IP transit.
+                        - interface_name cannot be updated.
+                        type: str
+                      external_connectivity_ip_pool_name:
+                        description:
+                        - IP address range allocated for communication between the SDA fabric and external networks.
+                        - Mandatory parameter for adding the L3 Handoff with IP transit.
+                        - This IP pool should be reserved in the fabric site.
+                        - If external_connectivity_ip_pool_name is set, then no need to set the local and remote addresses.
+                        - Setting external_connectivity_ip_pool_name will automatically set the local and remote adresses.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        - Updating the IP addresses are not permitted.
+                        type: str
+                      virtual_network_name:
+                        description:
+                        - Refers to the logical segmentation of the network, grouping devices into isolated virtual networks.
+                        - Either virtual_network_name or vlan_id is mandatory for all operations in L3 Handoff with IP transit.
```

Comment:
- Either `virtual_network_name` or `vlan_id` is required for all operations in L3 Handoff with IP transit.